src/TooltipPopoverWrapper.js

```diff
@@ -16,7 +16,7 @@ export const propTypes = {
   isOpen: PropTypes.bool,
   disabled: PropTypes.bool,
   hideArrow: PropTypes.bool,
-  boundariesElement: PropTypes.string,
+  boundariesElement: PropTypes.string, //PropTypes.oneOfType([PropTypes.string, DOMElement]),
```

Comment:
probably should be `PropTypes.oneOfType([PropTypes.string, DOMElement])`. `DOMElement` comes from `./utils` which is already referenced as an import and just need to be added as one of the named imports.